pages/styling-test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+    Heading,
+    Center,
+    Text,
+    VStack,
+    Button,
+    useDisclosure,
+} from "@chakra-ui/react";
```

Comment:
Let's move this file into a comment for this PR instead of merging it in - sorry if I wasn't clear during standup